FE/src/App.jsx

```diff
@@ -23,7 +27,16 @@ function App() {
       <Route path ="/recipes/:recipeId/edit" element={<EditRecipe token ={token}/>}></Route>
       <Route path ="/recipes/new" element={<NewRecipe token ={token}/>}></Route>
       <Route path ="/login" element={<Login token ={token} setToken ={setToken}/>}></Route>
-      <Route path ="/account" element={<Account token ={token}/>}></Route>
+      <Route path ="/account" element={<Account token ={token}/>}>
```

Comment:
On this page all you need is 
```js
<Route path ="/account/*" element={<Account token ={token}/>}></Route>
```

Then on the account page you'll also import Routes and Route. That would look something like this.

```js
// the rest of your component
<Routes>
   <Route path="/" element={<UserRecipes/>}/>
   <Route path="/recipes" element={<UserRecipes/>}/>
   <Route path="/reviews" element={<UserReviews/>}/>
   <Route path="/comments" element={<UserComments/>}/>
<Routes>

// the rest of your component

```